components/availability/src/init.spec.js

```diff
@@ -327,6 +341,7 @@ describe("availability component", () => {
       cy.get("nylas-availability").then((element) => {
         const component = element[0];
         component.allow_booking = true;
+        component.max_bookable_slots = 5;
```

Comment:
not sure why this is set to 5 when only 3 time slots are clicked in this test?

Reply:
Just wanted this to not be a blocker if we expanded this test so gave some buffer space. But yep! Can lower this to 3.